packages/filebrowser/src/listing.ts

```diff
@@ -941,6 +945,8 @@ export class DirListing extends Widget {
     const { width } =
       msg.width === -1 ? this.node.getBoundingClientRect() : msg;
     this.toggleClass('jp-DirListing-narrow', width < 250);
+    // Rerender item nodes, so that their modified dates update.
+    this.updateNodes(this._sortedItems, this._items);
```

Comment:
I would only re-render the last modified column instead. I would also bail if `this._modifiedWidth` has not crossed the boundary since the last time. Once this only does what it should I would consider putting it behind a throttler.